src/btree/bt_page.c

```diff
@@ -587,7 +591,9 @@ __inmem_row_leaf(WT_SESSION_IMPL *session, WT_PAGE *page, bool check_unstable)
              * Simple values without compression can be directly referenced on the page to avoid
              * repeatedly unpacking their cells.
              */
-            if (!btree->huffman_value)
+            if (!btree->huffman_value && unpack.stop_txn == WT_TXN_MAX &&
+              unpack.stop_ts == WT_TS_MAX && txn_global->has_oldest_timestamp &&
+              unpack.start_ts < txn_global->oldest_timestamp)
```

Comment:
same as the previous comment about comparison operation.
In my mental model, I always imagine start to use "<=" and stop to use "<" as start timepair is inclusive of validity and stop is exclusive of validity. If for any reason this logic is not valid in this case, I think it deserves a comment.

Reply:
Thanks! That's a good way to think of it.